src/explore-education-statistics-common/src/components/AccordionSection.module.scss

```diff
@@ -24,6 +17,20 @@
   }
 }
 
-.sectionContentExpanded {
-  @include accordion-section-expanded();
+.section {
+  @media print {
+    display: block !important;
+    page-break-inside: avoid;
+  }
+}
+
+.sectionContent {
+  display: block !important;
+  height: auto;
+  padding: 15px 0 !important;
+  visibility: visible;
+}
+
+.sectionContentCollapsed {
```

Comment:
Do we need this class? I'm struggling to see why Stu went to trouble of doing all of this stuff when there's already a `govuk-accordion__section--expanded` class.